docs/standards.md

```diff
@@ -0,0 +1,52 @@
+# Mapping Standards
+
+This document details the standards that all contributions to Parchment mappings must follow.
+
+### Parameter Names
+
+1. Use American English for spellings
+    - Example: `color`, not `colour`, `armor`, not `armour`
+2. All names should be prefixed with `p`
+    - Example: `pLevel`, `pXPos`
+3. Names should make sense without the prefix, and should be valid java identifiers NOT reserved keywords as such:
+    - Example: `pFloat` is NOT valid, as `float` is a reserved keyword.
+    - Example:`pOs` is not a correct use of the prefix, it should be `pPos`
+4. Parameter names should be based on current context but otherwise verbose and complete words
+5. Avoid single letters, or abbreviations.
+    - Example: `pNorthWest` over `pNW`, `pMatrixStack` over `pMStack`. 
+    - Exception: common abbreviations can be used: `IO` / `NBT`, etc.
+6. Use simple parameter names based off of class names, differentiating when there multiple of same type (using more words, not numbers)
```

Comment:
Doesn't 4 and 6 contradict each other? Were they both in the doc and we just forgot removing one or the other?

Reply:
They were both in the doc yes. I would agree they're not perfectly worded but disagree they're contradictory. i.e. `Level level` is both verbose (i.e. not `lvl`), a complete word, and based off the class names. Although I'm not sure how to reword these exactly to be more clear

Reply:
Combine them into the same point then. `Parameter names should be based off of the class names and the current context. They should be verbose and valid words, differentiating when there are multiple of the same type.`